polyfills/Array/from/tests.js

```diff
@@ -108,11 +108,37 @@ describe('returns an array with', function () {
 			proclaim.deepEqual(Array.from(iterator(3)), [3, 2, 1]);
 		});
 
+		it('can convert from a user-defined iterator using the Symbol.iterator', function () {
```

Comment:
Is there a case that this service doesn't provide Symbol polyfills even when browsers doesn't support the Symbol?